src/tests/CMakeLists.txt

```diff
@@ -3,7 +3,12 @@ list(APPEND test_cases test_edge_base)
 list(APPEND test_cases test_edgecontainer)
 list(APPEND test_cases test_elements)
 list(APPEND test_cases test_floatingpointcomparison)
+list(APPEND test_cases test_graph_base)
+list(APPEND test_cases test_graph_bf_visitor)
+list(APPEND test_cases test_graphalgorithm)
+list(APPEND test_cases test_graphdistvisitor)
 list(APPEND test_cases test_graphnode)
+list(APPEND test_cases test_graphvisitor)
```

Comment:
Huh, why are you doing this list append? I think all tests should just be listed in the `foreach()` statement.

Reply:
Because that turns into a really long single line.

Reply:
Unless you know of a way to break up such lines?

Reply:
Just do a line break, e.g.:
```cmake
foreach(i a b c
	d e f)
  message("${i}")
endforeach()
```

Reply:
Well that's easy enough 👍